pkg/operator/controller/controller_suite_test.go

```diff
@@ -86,7 +86,8 @@ var _ = BeforeSuite(func(done Done) {
 	Expect(err).NotTo(HaveOccurred())
 
 	opts := envtest.CRDInstallOptions{
-		CRDs: []*extv1beta1.CustomResourceDefinition{crd},
+		// TODO: Fix me before merge
```

Comment:
?

Reply:
So envtest package actually uses apiextensions v1beta1 in their code, but since we just moved to v1 this doesn't compile. Now obviously we still want this to work, so we need to do something to fix it. Maybe find a newer version of envtest that uses v1? I didn't want to forget or have it get lost in the shuffle so I put the TODO on here.

Reply:
Looks like controller runtime was updated to accept `runtime.Object` instead.  Maybe try updating CR?

Reply:
But I would not be sad if we get rid of the tests that require the `envtest` package

Reply:
So it looks like the controller-runtime > 0.4 requires a new version of go than we currently support.

Reply:
Looks like 0.5.0 also may require k8s 1.17 libraries as well.

I think envtest is only needed for one case that is not super important.  How about we disable for now and come back to it when we update go and k8s client libs?

Reply:
Sounds like a plan. I will put some comments in there explaining what is going on.